tests/bearlib/aspects/TasteTest.py

```diff
@@ -17,6 +17,7 @@ def test__init__defaults(self):
         assert taste.description is ''
         assert taste.suggested_values is ()
         assert taste.default is None
+        assert taste.languages in [(), ('C#',)]
```

Comment:
@Madhukar-SH the line you referencing is just docstring. The Taste code itself define only empty tuple `()` as default languages.